tutorials/networking/high_level_multiplayer.rst

```diff
@@ -371,9 +372,151 @@ have loaded the game scene.
         players.clear()
         server_disconnected.emit()
 
+ .. code-tab:: csharp C#
+    using Godot;
+    using Godot.Collections;
+
+    public partial class Lobby : Node
+    {
+        // These signals can be connected to by a UI lobby scene or the game scene.
+        [Signal]
+        public delegate void PlayerConnectedEventHandler(int peerId, Dictionary<string, string> playerInfo);
+
+        [Signal]
+        public delegate void PlayerDisconnectedEventHandler(int peerId);
+
+        [Signal]
+        public delegate void ServerDisconnectedEventHandler();
+
+        private const int _port = 7000;
+        private const string _defaultServerIp = "127.0.0.1";
+        private const int _maxConnections = 20;
+
+        // This will contain player info for every player, with the keys being each player's unique IDs.
+        private readonly Dictionary<long, Dictionary<string, string>> _players = new();
+
+        // This is the local player info. This should be modified locally before the connection is made.
+        // It will be passed to every other peer.
+        // For example, the value of "name" can be set to something the player entered in a UI scene.
+        private readonly Dictionary<string, string> _playerInfo = new()
```

Comment:
For Godot collections we usually fully-qualify them so it's easier to see that it's a Godot collection and not a System collection, since they share the same names it can be confusing otherwise.

```suggestion
        private readonly Godot.Collections.Dictionary<string, string> _playerInfo = new()
```